scripts/diff.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+'use strict'
+
+const fs = require('fs/promises')
+const path = require('path')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+require('make-promises-safe') // exit on error
+
+const DIST_ROOT = '/ipns/dist.ipfs.io'
+const CHANGE_TYPE_ADD = 0
+const CHANGE_TYPE_REMOVE = 1
+const CHANGE_TYPE_MOD = 2
+
+// Set of files to skip printing diffs.
+// These typically contain minified code that produce unreadable diffs.
+// Fail open so we don't accidentally miss something.
+const skipDiff = new Set(['site.js', 'site.css'])
```

Comment:
Yep, diffs of minified content are annoying. However, we probably want to at least know that these files are changed so that we can check on them when they're expected (e.g. bumping one of our JS deps) and verify they're not happening when unexpected.

Reply:
Good point, I'll add a list at the beginning of files that changed but whose diffs are not shown.